src/coredump/coredump-ratelimit.c

```diff
@@ -0,0 +1,170 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <sys/statvfs.h>
+#include "dirent-util.h"
+#include "fd-util.h"
+#include "fs-util.h"
+#include "string-util.h"
+#include "time-util.h"
+#include "user-util.h"
+#include "util.h"
+#include "coredump-ratelimit.h"
+#include "sd-id128.h"
```

Comment:
sd-id128 is one of our public headers (all headers starting with "sd-" are), i.e. one that external code that links against libsystemd.so also uses. We typically include them for our internal headers. i.e. please use:

```c
#include <....>
…

#include "sd-…"
…

#include "…"
…
```

Reply:
Addressed the comment. Please check once.